src/views/menus/NotificationsMenu.tsx

```diff
@@ -108,7 +108,7 @@ export const NotificationsMenu = ({
               },
             })),
         }))
-        .filter(({ items }) => items.length),
+        .filter(({ items: allItems }) => allItems.length),
```

Comment:
in cases like these i actually prefer using dot notation over destructuring. it's nice to have the property name be consistent throughout the file/codebase. anyone have thoughts on that? 🤔

Reply:
This can't be items because of shadowed variables - items is a variable in the upper scope.